tools/code_format/glint.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+NOLINT_RE="\.patch$|^test/server/server_corpus/clusterfuzz|test/extensions/transport_sockets/tls/test_data/password_protected_password.txt$|Whitespace.txt$|WhiteSpace.txt$"
```

Comment:
@justin-mp do you know if the trailing whitespace is necessary for these tests' data (ie not all of the ws)

Reply:
re these `Whitespace.txt$|WhiteSpace.txt$"`

Reply:
All of the trailing whitespace in `test/extensions/filters/http/cdn_loop/parser_corpus` is intentional since there are weird edge cases that the parser is trying to get at.  (Some of them, for example, are even in the RFC!)

Reply:
ok - ill revert this dir and exclude

Reply:
thanks @justin-mp